src/fe-text/irssi.c

```diff
@@ -352,6 +353,7 @@ int main(int argc, char **argv)
 	   you have to call setlocale(LC_ALL, "") */
 	setlocale(LC_ALL, "");
 
+	loglev = g_log_set_always_fatal(G_LOG_FATAL_MASK | G_LOG_LEVEL_CRITICAL);
```

Comment:
There's also some `s/g_error/g_critical/` - maybe could reverse those, have all of them as `g_error` and get rid of this `g_log_set_always_fatal`?